src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+              Collaborate with your remote team to improve what you do, guided
+              by a simple workflow. And for free!
+            </Text>
+          </Box>
+          <Flex mb={2} justifyContent="center" textAlign="center">
+            {/*
+            // @ts-ignore */}
+            <Button
+              as="a"
+              href="/new"
+              variant="landingPrimary"
+              fontSize={[3, null, 4]}
+              tabIndex={1}
+              px={[4, null, 7]}
+            >
+              Create a new retro
+            </Button>
+            {/*
+            // @ts-ignore */}
+            <Button
+              as="a"
+              href="#how-it-works"
+              variant="landingSecondary"
+              ml={5}
+              tabIndex={2}
+              fontSize={[3, null, 4]}
+              px={[4, null, 7]}
+            >
+              See how it works
+            </Button>
+          </Flex>
+          <Text
+            fontSize={[3, null, 4]}
+            color="grey"
+            mt={7}
+            mb={[4, null, null, 0]}
+          >
+            Already{" "}
+            <Text fontSize="inherit" color="dark" as="span" fontWeight="bold">
+              1.259 retros
+            </Text>{" "}
+            and counting.
+          </Text>
+        </Section>
+        <Section bg="violet" color="white" py={[8, null, null, 80, 100]} px={7}>
+          <Box maxWidth="100ch" margin="auto">
+            <Text as="h2" hidden>
+              Main features
+            </Text>
+            <Flex justifyContent="space-between" flexWrap="wrap">
+              <Box
+                maxWidth={["100%", null, "45%", "20%"]}
+                mb={[9, 0]}
+                height={["auto", 250, "auto"]}
+              >
+                <Mobile height={32} />
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  Easy peasy
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  A mobile friendly UI with a clean and fast workflow that
+                  doesn't even need a tutorial.
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  Did we say already that it's free?
+                </Text>
+              </Box>
+              <Box
+                maxWidth={["100%", null, "45%", "20%"]}
+                mb={[9, 0]}
+                height={["auto", "auto", 260]}
+              >
+                <LocationOff height={32} />
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  Anonymous
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  We don't track or store any personal information.
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  And the code is open source so you can always check what we
```

Comment:
I'd remove the first `And`, it's a bad habit to start sentences with a conjunction